README.md

```diff
@@ -28,3 +28,15 @@ Deployment is automated.
 Commits to default branch will trigger a new build to the primary domain.
 
 PR's each receive their own preview URL ahead of deployment.
+
+### Mermaid Diagrams
+
+To insert mermaid diagrams, use the following code blocks in .md or .mdx
+
+```mermaid
+graph TD;
+    A-->B;
+    A-->C;
+    B-->D;
+    C-->D;
+```
```

Comment:
```suggestion
```
This can be dropped from here, but would be excellent to add to https://github.com/PlaceOS/docs#extended-syntax.